recipe/meta.yaml

```diff
@@ -31,8 +31,8 @@ requirements:
     - tornado
     - requests
     - jsonschema
-    - pyqt
-    - pyqtgraph
+    - pyqt 4.11.*
+    - pyqt 4.11.*graph
```

Comment:
Its probably better to use some regular expression to deal with trailing whitespaces and what not.